server/app.js

```diff
@@ -11,6 +11,18 @@ require( 'dotenv' ).load();
 var Logger = require( './components/Logger' );
 Logger.initialize();
 
+if( true ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Merge this if statement with the nested one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube:9000/coding_rules#rule_key=javascript%3ACollapsibleIfStatements)